website/content/docs/install/performance.mdx

```diff
@@ -1,10 +1,8 @@
 ---
 layout: docs
-page_title: Server Performance
+page_title: Server Requirements
```

Comment:
Not sure about this change. It is kinda specific to performance, it's not about generic server requirements. Options I like:
* Server Performance
* Server Performance Tuning
* Server Performance Requirements

Reply:
I like "Server Performance Requirements"

It's not about the performance of servers, or the general requirements for servers. It's about the requirements needed for performant operation.

Reply:
+1 to server performance requirements